packages/twenty-server/src/engine/integrations/llm-prompt-template/interfaces/llm-prompt-template-name.interface.ts

```diff
@@ -0,0 +1,3 @@
+export enum LLMPromptTemplateEnvVar {
+  AskAI = 'PROMPT_TEMPLATE_NAME_ASK_AI',
```

Comment:
As mentioned previously, it's not clear for an external reader what this variable will really be useful for